controllers/pvc.go

```diff
@@ -187,7 +188,7 @@ func (r *VolumeSnapshotBackupReconciler) BindPVCToDummyPod(log logr.Logger) (boo
 			Containers: []corev1.Container{
 				{
 					Name:  "busybox",
-					Image: DummyPodImage,
+					Image: getDataMoverDummyPodImage,
```

Comment:
```suggestion
					Image: getDataMoverDummyPodImage(),
```

Reply:
Thanks, fixed. I made the same error on the operator and hit it testing. Fixed there as well.